### ISSUE TITLE
Minimal Frontend

### ISSUE THREAD

**Issue by MatthewMerrill** [MatthewMerrill/AttendOMatic#2]

**Comment by MatthewMerrill** [comment]
With an HTML <form>, you should be able to send a POST to the /api/attending route with 2-3 HTML attributes.

Check out action and method
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form

**Comment by dipamin324** [comment]
<html>
<head>
<title> AttendOMatic </title>
</head>
<body>

<h1 style = font-family:Comic Sans Ms; text-align ="center"; font-size:20pt;
color:#00FF00;>
    AttendOMatic
    </h1>

```
<form name = "login">
ID # <input type="text" name="userid"/>
<input type = "button" onclick=“check(this.form)” value="Submit"/>
<input type = "reset" value = "Reset"/>
</form>
<script language="javascript">
function check(form)
//Checks ID #
{
//below checks if ID # matches
if(form.userid.value == “ID”){
    window.open('SheetsQuickstart.java')
    } else {
        alert("ID # does not exist")
    }
</script>
</body>
</html>
```

Got this so far, can't figure out how to tie it in with the quicksheets so when the login in is successful it runs. Any help would be appreciated

**Comment by dipamin324** [comment]
``` HTML
<html>
<head>
<title> AttendOMatic </title>
</head>
<body>
<h1 style = font-family:Comic Sans Ms; text-align ="center"; font-size:20pt;
color:#00FF00;>
    AttendOMatic
    </h1>
    <form name = "login">
    ID # <input type="text" name="userid"/>
    <input type = "button" onclick=“check(this.form)” value="Submit"/>
    <input type = "reset" value = "Reset"/>
    </form>
    <script language="javascript">
    function check(form)
    //Checks ID #
    {
    //below checks if ID # matches
    if(form.userid.value == “ID”){
        window.open('SheetsQuickstart.java')
        } else {
            alert("ID # does not exist")
        }
    </script>
    </body>
    </html>
```

**Comment by dipamin324** [comment]
idk why the top <html> part is getting cut off but here is the txt file
[login.txt](https://github.com/MatthewMerrill/AttendOMatic/files/504093/login.txt)

**Comment by MatthewMerrill** [comment]
GitHub comments allow inline HTML, so it was trying to use parts of your code for the comment itself. You can tell it to format any section of a comment as code by surrounding it with "```" (three of the key to the left of your "1" key) followed by the type of your code. For example,

```Java
public static void main(String args){
&nbsp;&nbsp;&nbsp;&nbsp;System.out.println("Hello World");
}
```

Becomes

``` Java
public static void main(String[] args){
    System.out.println("Hello World);
}
```

**Comment by MatthewMerrill** [comment]
Patched in: e8e32ad3c791c520abe9a237fd747b52b24e2d93
